client/requirements.in

```diff
@@ -0,0 +1,16 @@
+ray[default,data]>2.6<3
```

Comment:
I was getting errors when this was unpinned. One requirements file would install ray 2.3 and one would install ray 2.6, which led to conflicts between the dev and production envs.